### PR TITLE
16 feature register page

### PR DESCRIPTION
**Pull request by luke-welbourn**
allows the user to sign up for an account and provides details in a user friendly clear form.

if the account creation is successful the user is redirected to the home page.

### CODE REVIEW

File: src/pages/Register/Register.tsx [hunk 1]
```diff
@@ -38,14 +51,158 @@ const Register = ({
...
+  };
+
+  const toggle = () => {
+    if (userinput) {
+      setUserInput(false);
+    } else {
+      setUserInput(true);
+    }
+  };
+
```
Comment (olirob93) [inline src/pages/Register/Register.tsx:110]: can't you just change the toggle function to be called handleClickNext as all handleClickNext is doing is running the toggle function?

You can then just replace the one instance of where you are using the toggle function on and arrow image to handleClickNext.

**Review by olirob93 — CHANGES_REQUESTED** [review]
just a small change

**Review by olirob93 — APPROVED** [review]
nice looks good

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

**Review by Charlie-robin — APPROVED** [review]
Looks good to me

**Pull request merged**